library/src/androidTest/java/com/owncloud/android/GetCapabilitiesIT.java

```diff
@@ -145,5 +145,16 @@ private void checkCapability(OCCapability capability) {
             // files_lock app needs to be installed in server for this to work
             assertNotNull(capability.getFilesLockingVersion());
         }
+
+        // groupfolder
+        if (capability.getVersion().isNewerOrEqual(NextcloudVersion.nextcloud_26)) {
```

Comment:
Shipped with Nc26?

Reply:
Way too fast releases, thus mixed it up.
Will change to NC27.